packages/web-frontend/src/components/View/ChartWrapper/ReferenceLabel.js

```diff
@@ -7,14 +7,21 @@
 
 import React from 'react';
 
+function returnTrueIfMax(value) {
+  if (value.includes('MAX') || value.includes('Max') || value.includes('max')) return true;
+  return false;
+}
+
 function ReferenceLabel(props) {
   const { fill, fontSize, value, viewBox } = props;
   const x = viewBox.width / 2 + 30;
-  const y = viewBox.y + 15;
+  const defaultY = viewBox.y + 15;
+  const maxY = viewBox.y - 5;
+  const getYPosition = returnTrueIfMax(value) ? maxY : defaultY;
 
   if (value == null) return null;
   return (
-    <text x={x} y={y} fill={fill} fontSize={fontSize || 14} fontWeight="bolder">
+    <text x={x} y={getYPosition} fill={fill} fontSize={fontSize || 14} fontWeight="bolder">
```

Comment:
Side note, `getX`, to me, implies that this is a function, whereas it is just a value in this case.